docs/development/release-process.rst

```diff
@@ -314,3 +314,15 @@ We don't do maintenance / patch releases, including for security issues, due to
 We just do a normal release using the calendar base versioning scheme.
 
 We welcome additional volunteers to help drive the release effort.
+
+
+Notes
+-----
+
+The release process uses a GitHub Actions environment, configured here: 
+https://github.com/twisted/twisted/settings/environments/4731362866/edit
+currently only tags of the form 'twisted-*' can use that environment
+and only CI runs in that environment can release to PyPI.
+
+In the future it could be possible to add collaborators who can, for example,
```

Comment:
I don't know what this means.

I think it's important to note that that the `test.yaml` GitHub Worflow is approved to trigger a release for PyPI.

If in the future we do a release from a workflow with a different name, we need to update the PyPI security configuration.

Reply:
But that doesn't work with the all-successful pattern, we should keep this test.yaml forever. If we need to split up the file we can import workflows into it

Reply:
Ah ok, well do you want to add a screenshot of the config here?

Reply:
I don't think that we should add the screenshot to the docs.

It's enough to have the screenshot on the PR, as a snapshot of the configuration at the time this was merged